props/prop_base.h

```diff
@@ -504,7 +504,10 @@ class PropBase : CommandParser, Looper, protected SaberBase {
 #ifdef SAVE_PRESET
     SaveState(current_preset_.preset_num + 1);
 #endif
+
+#ifndef DISABLE_FAST_PRESET_CHANGE
```

Comment:
This breaks the next_preset_fast() contract, and is not the right way to disable functionality.  If you don't want to go to the next preset, then don't call this function.